torch/ao/sparsity/experimental/data_sparsifier/data_norm_sparsifier.py

```diff
@@ -80,7 +80,7 @@ def __get_block_level_mask(self, data,
         mask = self.__get_scatter_folded_mask(data=unfolded_data, dim=1, indices=sorted_idx, output_size=padded_data.shape,
                                               sparse_block_shape=sparse_block_shape)
 
-        mask = mask.squeeze()[:height, :width].contiguous()  # remove padding and make contiguous
+        mask = mask.squeeze(0).squeeze(0)[:height, :width].contiguous()  # remove padding and make contiguous
```

Comment:
Is this what you want? Calling squeeze(0) twice only works if the first dim is 1. Maybe you mean `.squeeze(1).squeeze(0)`?

Example:

1. For `mask.shape = 1 x 1 x A x B`, we have:
    - `mask.squeeze(0).squeeze(0)` produces `A x B` 
    - `mask.squeeze(1).squeeze(0)` produces `A x B` 
2. For `mask.shape = X x 1 x A x B`, we have:
    - `mask.squeeze(0).squeeze(0)` produces `X x 1 x A x B` 
    - `mask.squeeze(1).squeeze(0)` produces `X x A x B`

Reply:
Ideally, this works for 2D data. If the data is 1D then we make it unsqueeze and proceed the same way. The mask that we get after scatter folding is 4D, so we squeeze it two times to make it 2D back again.